components/brave_rewards/resources/shared/components/onboarding/css_mixins.ts

```diff
@@ -15,8 +15,8 @@ export const enableRewardsButton = `
   line-height: 20px;
   cursor: pointer;
 
-  &:active {
-    background: var(--brave-color-brandBatActive);
+  &:focus-visible {
+    outline: none;
```

Comment:
Is this OK from an accessibility perspective? I don't know - simply asking.

Reply:
As discussed, removed auto-focus and left the ring as is.